### PR TITLE
[CDF-516] - Migrate xmlaQueries as a requireJS module

### PR DESCRIPTION
**Pull request by afrjorge**
- created AMD module for XML/A queries
	- created samples for MDX, XML/A and XML/A discovery queries
	- created unit test for XML/A queries

### CODE REVIEW

**Comment by afrjorge** [comment]
@pamval please review, tested with BI Server EE 5.1.2.0 and CE 5.3.0.0

**Comment by afrjorge** [comment]
@pamval please review

File: cdf-core/cdf/js-modules/queries/XmlaQuery.js [hunk 1]
```diff
@@ -0,0 +1,271 @@
+/*!
```
Comment (pamval) [inline cdf-core/cdf/js-modules/queries/XmlaQuery.js]: This class needs to be documented

**Comment by afrjorge** [comment]
@pamval please review

**Comment by afrjorge** [comment]
@pamval please review

**Pull request merged**